pkg/appgw/requestroutingrules.go

```diff
@@ -227,3 +242,31 @@ func (builder *appGwConfigBuilder) RequestRoutingRules(ingressList [](*v1beta1.I
 	builder.appGwConfig.URLPathMaps = &urlPathMapFiltered
 	return builder, nil
 }
+
+func (builder *appGwConfigBuilder) getSslRedirectConfigResourceReference(ingress *v1beta1.Ingress) *network.SubResource {
+	configName := generateSSLRedirectConfigurationName(ingress.Namespace, ingress.Name)
+	sslRedirectConfigID := builder.appGwIdentifier.redirectConfigurationID(configName)
+	return resourceRef(sslRedirectConfigID)
+}
+
+func (builder *appGwConfigBuilder) modifyPathRulesForRedirection(ingress *v1beta1.Ingress, httpURLPathMap *network.ApplicationGatewayURLPathMap) {
+	// Application Gateway supports Basic and Path-based rules
+
+	if len(*httpURLPathMap.PathRules) == 0 {
+		// There are no paths. This is a rule of type "Basic"
+		redirectRef := builder.getSslRedirectConfigResourceReference(ingress)
+		glog.Infof("Attaching redirection config %s to basic request routing rule: %s\n", *redirectRef.ID, *httpURLPathMap.Name)
+		httpURLPathMap.DefaultRedirectConfiguration = redirectRef
```

Comment:
Can you set default backend pool and http setting to nil?

Reply:
Good point! Done!